Extend list of DOM methods

Zirro opened the pull request:
This pull request extends the list of methods recognized as being part of the DOM. The original list was very incomplete and led to peculiar behaviour such as classList.add() and classList.remove() being coloured differently.

These additions were collected by iterating through several common DOM interfaces, mainly in Firefox and Chrome. For practical reasons it is by no means complete, but I believe it covers those which are most-used. Intentional omissions include prefixed properties and non-standard properties with little adoption. I also removed a couple of overlapping properties from the support.function.js regex.

50Wliu commented:
For stuff like `(exec|execScript)`, that won't work and will exit as soon as it reaches `exec`.  So you have two options:
1. `(exec(script)?)`
2. `(execScript|exec)`

Zirro commented:
While I moved execCommand() in this PR, I don't think I changed the (exec|execScript)-logic. Furthermore, .exec(), .execScript() and .execCommand() all seem to highlight and receive the correct classes when I test in Atom.

50Wliu commented:
Hmm, you're right - sorry about that.  Just waiting for CI at this point.

50Wliu commented:
Don't know what's up with Travis right now -.-
